frontend/app/src/components/home/profile/LinkedAuthAccounts.svelte

```diff
@@ -57,6 +71,19 @@
                     </div>
                 </TooltipWrapper>
             </div>
+        {:else}
+            <div class="unlink">
+                <TooltipWrapper position="top" align="end">
+                    <Button slot="target" cls="btn" hollow on:click={() => (unlinking = account)}>
```

Comment:
I think this would be more consistent with the existing UI if we simply used the `HoverIcon` component. e.g. 

```
<HoverIcon slot="target" onclick={() => (unlinking = account)}>
    <LinkOff size={$iconSize} color={"var(--icon-txt)"} />
</HoverIcon>
```